unified-runtime/test/adapters/cuda/CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ add_adapter_test(cuda
         kernel_tests.cpp
         memory_tests.cpp
         event_tests.cpp
+        helpers.cpp
         #FIXME: make this cleaner
         ${CMAKE_CURRENT_SOURCE_DIR}/../../../source/adapters/cuda/queue.cpp
         ${CMAKE_CURRENT_SOURCE_DIR}/../../../source/adapters/cuda/common.cpp
```

Comment:
```suggestion
        ${CMAKE_CURRENT_SOURCE_DIR}/../../../source/adapters/cuda/common.cpp
        ${CMAKE_CURRENT_SOURCE_DIR}/../../../source/adapters/cuda/usm.cpp
```
Would this work? I don't really like copy-pasting the definition in the tests. Maybe we could also copy it from `usm.cpp` to `common.cpp` which is already included.

Reply:
I tried `usm.cpp`, but that started complaining about other symbols being undefined so it's probably going to spiral out of control. I can try moving it to `common.cpp` though.

Reply:
Moving it to common.cpp seems to work.